java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -8213,6 +8246,17 @@ final public void addLayoutTrack(@Nonnull LayoutTrack trk, @Nonnull LayoutTrackV
         viewToTrk.put(v, trk);
 
         unionToPanelBounds(v.getBounds()); // temporary - this should probably _not_ be in the topological part
+
+        // Add tooltips to layout turnouts
+        if (trk instanceof LayoutTurnout) {
+            var lt = (LayoutTurnout) trk;
+            var tt = new ToolTip(null, 0, 0, new Font("SansSerif", Font.PLAIN, 12),
+                Color.black, new Color(215, 225, 255), Color.black, null);
+            lt.setToolTip(tt);
+            lt.setShowToolTip(showToolTip());
+            log.debug("Layout turnout tooltip added: {}", lt);
+        }
```

Comment:
I suggest adding this section to the constructor of `LayoutTurnout`.

Reply:
Good idea.  Done.